docs/source/index.rst

```diff
@@ -1,32 +1,64 @@
+
+.. _Umbral: https://github.com/nucypher/pyUmbral
+
+.. image:: ./.static/img/threshold_logo_purple.png
+    :target: ./.static/img/threshold_logo_purple.png
+
+
+`Welcome <https://blog.threshold.network/threshold-launch/>`_ to the Threshold Network!
+
 Introduction
 =============
 
+The `NuCypher Network <https://www.nucypher.com/>`_ has merged with the
+`KEEP Network <https://keep.network/>`_ to form the `Threshold Network <https://threshold.network/>`_!
 
-.. _Umbral: https://github.com/nucypher/pyUmbral
+The Threshold Network powers user sovereignty on the public blockchain. It provides a decentralized
+network of nodes that perform threshold cryptography operations as a service to ensure full control over
+your digital assets.
 
+Proxy Re-encryption (PRE) Application
+=====================================
 
-The NuCypher network is a decentralized network of nodes that perform threshold
-cryptography operations serving users with secrets management and dynamic access control.
-``nucypher`` is the python library and CLI for interacting with the
-decentralized threshold cryptography network.
+The NuCypher Network proxy re-encryption (PRE) service is now the PRE Application on the Threshold Network.
 
+The PRE Application is the first of many *threshold cryptography*-based applications
+to be hosted on the Threshold Network. PRE is an end-to-end encryption protocol that is a more scalable, more flexible form of
+`public-key encryption <https://en.wikipedia.org/wiki/Public-key_cryptography>`_ and enables a group of proxy
+entities to transform encrypted data from one public key to another, without the power to decrypt the data or gain access to any
+private keys. PRE equips developers, applications and end-users with **secrets management** and **dynamic access control** capabilities.
+This service is provided by a decentralized array of nodes on the Threshold Network, each running the same PRE client software.
 
-How does NuCypher work?
------------------------
+Private data, wherever stored, remains private, confidential
+and encrypted while maintaining the ability to share that data with trusted parties.
+
+``nucypher`` is the python library and CLI for interacting with the PRE Application.
```

Comment:
I'd say the angle has shifted so that ``nucypher`` is more about running PRE nodes than a library or user-facing client. In retrospect, I think the original intention of this sentence is lost, since it seemed to be a disambiguation between NuCypher (the network) and ``nucypher`` (the client), so I'd just remove it.

Reply:
✔️